test/extensions/tracers/xray/util_test.cc

```diff
@@ -0,0 +1,103 @@
+#include "extensions/tracers/xray/util.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace XRay {
+
+TEST(XRayUtilTest, WildcardMatchingInvalidArgs) {
+  const std::string pattern;
+  const std::string text = "whatever";
+  ASSERT_FALSE(WildcardMatch(pattern, text));
+}
+
+TEST(XRayUtilTest, WildcardMatchExactPositive) {
+  const std::string pattern = "foo";
+  const std::string text = "foo";
```

Comment:
why do you break out pattern and test into separate lines? Having one line per testcase seems easier to read.

I worked on an OSS wildcard matcher sometime back, and I have some testcases in there I'm not sure you've covered. Can you take a quick look at these?

https://github.com/apache/incubator-pagespeed-mod/blob/master/pagespeed/kernel/base/wildcard_test.cc

Reply:
yeah that's dumb, I'll change it. I figured the tests from the older PR were working so I copy pasted them.